src/extension.ts

```diff
@@ -147,22 +145,37 @@ class RunLineInTerminal {
     }
 }
 
+const elegantSpinner = require('elegant-spinner');
```

Comment:
Move to import section at the top of the file and use `import` instead of `require`.

Reply:
Looks like this will require me to create a new .d.ts file. Do you have any preference where that file should go?

Reply:
The latest version of elegant-spinner comes with a .d.ts. Have you tried that?

Reply:
I did:
`C:\Users\mburleigh\Source\Repos\vscode-azurecli-mfb>npm install @types/elegant-spinner
npm WARN deprecated @types/elegant-spinner@2.0.1: This is a stub types definition. elegant-spinner provides its own type definitions, so you do not need this installed.
+ @types/elegant-spinner@2.0.1
added 1 package in 2.515s`
after that looking in the \node_modules\@types\elegant-spinner directory there is no index.d.ts file.

Reply:
You don't need the @types package, but the 2.0 release of the main package from what I see.

Reply:
I'll take a look; thanks for the feedback!

Reply:
Fixed. Had to uninstall, clean up my node_modules, & reinstall to get the new version.